### PR TITLE
[8.0] ADD account_invoice_fiscal_agent

### PR DESCRIPTION
**Pull request by eLBati**
Account Fiscal Agent
====================

This module automates accounting entries of fiscal agent companies.

Use case
--------

Company A is located in country A.
Company B is located in country B and is the fiscal agent of company A, for country B.
When company A issues an invoice to customer C (of country B), company B has to manage fiscal matters on behalf of Company A.

This module allows you to configure fiscal agent accounts, journals, and taxes in fiscal positions of company A. So that, upon validating the invoice of company A, a new invoice (for company B) is created in the name of same partner with the above pre-configured details.

Usage
=====

To use this module, you need to:

* Map fiscal agent accounts, journals and taxes in the Fiscal Position. This must be done by a user who can see data from both the companies

### CODE REVIEW

**Review by moylop260 — APPROVED** [review]

**Review by tafaRU — CHANGES_REQUESTED** [review]
Please use UpperCamelCase for naming class as  
written at https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#odoo-python-classes

**Comment by eLBati** [comment]
@tafaRU done

**Review by tafaRU — APPROVED** [review]

**Comment by rafaelbn** [comment]
This PR has 2 approvals from more that 1 year so why it wasn't merged? 

ping @OCA/accounting-maintainers

**Pull request closed without merging**
